wiki/Beatmap_Editor/Compose/en.md

```diff
@@ -1,406 +1,355 @@
-<img src="Compose.png" title="fig:Compose editor for osu! in Windowed mode" alt="Compose editor for osu! in Windowed mode" width="200" /> <img src="ComposeM.jpg" title="fig:Compose for osu!mania using osu!screenshot (only the external upper menu is not shown)" alt="Compose for osu!mania using osu!screenshot (only the external upper menu is not shown)" width="200" />
+Compose
+==========
 
-**Compose** is the place where a [mapper](mapper "wikilink") spent most of the time mapping out his/her design with respect to timeline after setting the BPM and Timing Sections. This is also the place where players can practice a ranked difficulty and inspect its design akin to DDR's Training mode.
+![Compose editor for osu! in Windowed mode](Compose.png "Compose editor for osu! in Windowed mode")
+
+![Compose for osu!mania using osu!screenshot (only the external upper menu is not shown)](ComposeM.jpg "Compose for osu!mania using osu!screenshot (only the external upper menu is not shown)")
+
+**Compose** is the place where a [mapper](/wiki/Glossary) spent most of the time mapping out his/her design with respect to timeline after setting the BPM and Timing Sections. This is also the place where players can practice a ranked difficulty and inspect its design akin to DDR's Training mode.
 
 Due to different needs in mapping style (osu! is very flexible, allowing much room for creativity while osu!mania is restrained to selected K, demanding extreme precision to evoke the keysounds harmonically), osu!'s editor and osu!mania-only's editor are given their own Features section. You can easily access osu!mania editor by setting your difficulty to osu!mania-only.
```

Comment:
explain what **K** means --> ...`selected key (or _K_ for short), demanding`...
also, lowercase the `F` in `Features`